src/auxsolvers/auxsolvers.cpp

```diff
@@ -22,7 +24,9 @@ AuxSolvers::Solve(double t)
 {
   for (auto & auxsolver : _aux_queue)
   {
+    spdlog::stopwatch sw;
     auxsolver->Solve(t);
+    logger.info("AuxSolver Solve: {} seconds", sw);
```

Comment:
It would be particularly useful if this log line could echo the auxsolver name before and after the auxsolver solve (I understand that may require an extra member of auxsolvers to store names in the order of the auxqueue)

Reply:
Done! I changed _aux_queue to be a vector of pairs so that the auxsolver names can be sorted and preserved with the auxsolvers themselves